BookingsApi/BookingsApi.DAL/Migrations/20230704125913_UpdateVenuNamesHearingVenu.cs

```diff
@@ -0,0 +1,98 @@
+            { "Manchester Employment Tribunal",new Tuple<string, string>("Manchester Tribunal Hearing Centre - Alexandra House", "301017") },
+            { "Medway Magistrates Court and Family Court",new Tuple<string, string>("Medway Magistrates Court", "771467") },
+            { "Mold Justice Centre (Mold Law Courts)",new Tuple<string, string>("Mold Justice Centre", "211138") },
+            { "Newton Aycliffe Magistrates Court",new Tuple<string, string>("Newton Aycliffe Magistrates Court and Family Court", "659436") },
+            { "Northampton Crown, County and Family Court",new Tuple<string, string>("Northampton Crown Court, County Court and Family Court", "195489") },
+            { "Oxford Magistrates Court",new Tuple<string, string>("Oxford and Southern Oxfordshire Magistrates Court", "732661") },
+            { "Plymouth (St Catherine's House)",new Tuple<string, string>("Plymouth As St Catherine's House", "235590") },
+            { "Preston Crown Court and Family Court (Sessions House)",new Tuple<string, string>("Preston Crown Court", "102476") },
+            { "Reedley Family Hearing Centre",new Tuple<string, string>("Reedley Magistrates Court and Family Court", "739294") },
+            { "Sheffield Designated Family Court",new Tuple<string, string>("Sheffield Family Hearing Centre", "778638") },
+            { "Shrewsbury Crown Court",new Tuple<string, string>("Shrewsbury Justice Centre", "259170") },
+            { "Skipton County Court and Family Court",new Tuple<string, string>("Skipton Magistrates and County Court", "318324") },
+            { "South Tyneside County Court and Family Court",new Tuple<string, string>("South Tyneside Magistrates Court and Family Court", "563156") },
+            { "Southend Crown Court",new Tuple<string, string>("Southend Combined - Crown, Mags, County and Family Court", "781139") },
+            { "Stockport Magistrates Court",new Tuple<string, string>("Stockport Magistrates Court and Famiy Court", "560788") },
+            { "Swansea Civil Justice Centre",new Tuple<string, string>("Swansea Civil and Family Justice Centre", "234946") },
+            { "West Hampshire (Southampton) Magistrates Court",new Tuple<string, string>("West Hampshire Magistrates Court", "330480") },
+            { "Wolverhampton Social Security and Child Support Tribunal",new Tuple<string, string>("Wolverhampton Ast - Norwich Union House, Wolverhampton", "788436") },
+            { "Worcester Justice Centre",new Tuple<string, string>("Worcester Magistrates Court", "703200") },
+            { "Worthing County Court and Family Court",new Tuple<string, string>("Worthing Magistrates and County Court", "493880") },
+            { "Wrexham County and Family Court",new Tuple<string, string>("Wrexham Law Courts", "637145") },
+            { "Hereford Magistrates Court",new Tuple<string, string>("Hereford Justice Centre", "228883") },
```

Comment:
I don't this should be here - there is already another venue called Hereford Justice Centre, so you'll get 2 venues with the same name

Reply:
fixed